src/test/java/Homework18.java

```diff
@@ -6,7 +6,7 @@
 
 import java.time.Duration;
 
-public class Homework18 extends LoginTests {
+public class Homework18 extends BaseTest {
```

Comment:
Please, add assert.
_Good practice: one assert - one test_

For example 

![image](https://user-images.githubusercontent.com/104761482/222059094-41c15256-95bf-415e-a582-4a3d5abc93d8.png)